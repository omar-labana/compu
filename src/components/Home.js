import React from 'react';

const Home = () => (
  <main>
    <h2 className="text-2xl mt-2">Welcome to our page!</h2>
    <p className="text-2xl mt-2">
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
    </p>
    <p className="text-2xl mt-2">
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
    </p>
  </main>
);

export default Home;
