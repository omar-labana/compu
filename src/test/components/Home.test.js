import React from 'react';
import '@testing-library/jest-dom';
import Home from '../../components/Home';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

const stl = () => 'red';
const doNothing = () => ''

describe('Home Component', () => {
    let wrapper;
    wrapper = shallow(<Home buttonName="1" st={stl} clickHandler={doNothing} />);

    it('exists', () => {
        expect(wrapper.find('Home')).toBeTruthy();
    });

    it('has to match snap', () => {
        const home = renderer.create(<Home />);
        expect(home).toMatchSnapshot();
    });
})