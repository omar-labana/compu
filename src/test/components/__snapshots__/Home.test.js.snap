// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component has to match snap 1`] = `
<main>
  <h2
    className="text-2xl mt-2"
  >
    Welcome to our page!
  </h2>
  <p
    className="text-2xl mt-2"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
  </p>
  <p
    className="text-2xl mt-2"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
  </p>
</main>
`;
